app/desktop/studio_server/provider_api.py

```diff
@@ -170,6 +238,11 @@ async def get_available_models() -> List[AvailableModels]:
         if ollama_models:
             models.insert(0, ollama_models)
 
+        # Docker Model Runner is special: check which models are installed
+        docker_models = await available_docker_model_runner_models()
+        if docker_models:
+            models.insert(-1 if ollama_models else 0, docker_models)
```

Comment:
nit: -1 is a bit strange (before last item). Did you mean 1?

Maybe just add docker first at 0, and Ollama second  at 0, so Ollama keeps top spot if both are present?